SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DAQmx/Types.cs

```diff
@@ -109,5 +117,12 @@ public class DAQmxTimingSampleClockSettings
         /// This value is used to get/set the value of the NationalInstruments.DAQmx.Timing.SampleClockSource driver property.
         /// </remarks>
         public int? SamplesPerChannel { get; set; }
+
+        /// <summary>
+        /// Configures the timing settings for a DAQmx task, enabling the user to define one or more properties in a single operation.
```

Comment:
Please update the wording in a similar way as AOFunctionGenerationSettings.

Reply:
Updated